code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -0,0 +1,254 @@
+#define RUST_CULTIST 0
+#define FLESH_CULTIST 1
+#define ASH_CULTIST 2
+
+
+/proc/isecultist(mob/living/M)
+	return M && M.mind && M.mind.has_antag_datum(/datum/antagonist/ecult)
+
+/datum/antagonist/ecult
+	name = "Eldritch Cultist"
+	roundend_category = "eldritch cultist"
+	antagpanel_category = "Eldritch Cult"
+	antag_moodlet = /datum/mood_event/ecult
+	job_rank = ROLE_ECULTIST
+	var/ignore_implant = FALSE
+	var/give_equipment = TRUE
+	var/datum/team/ecult/ecult_team
+	var/list/researched_knowledge = list()
+	var/total_sacrifices = 0
+
+/datum/antagonist/ecult/get_team()
+	return ecult_team
+
+/datum/antagonist/ecult/admin_add(datum/mind/new_owner,mob/admin)
+	give_equipment = FALSE
+	new_owner.add_antag_datum(src)
+	message_admins("[key_name_admin(admin)] has cult'ed [key_name_admin(new_owner)].")
+	log_admin("[key_name(admin)] has cult'ed [key_name(new_owner)].")
+
+
+/datum/antagonist/ecult/create_team(datum/team/cult/new_team)
+	if(!new_team)
+		ecult_team = new /datum/team/ecult
+		return
+	if(!istype(new_team))
+		stack_trace("Wrong team type passed to [type] initialization.")
+	ecult_team = new_team
+
+/datum/antagonist/ecult/proc/add_objectives()
+	objectives |= ecult_team.objectives
+
+/datum/antagonist/ecult/greet()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ecult_op.ogg', 100, FALSE, pressure_affected = FALSE)//subject to change
+	to_chat(owner, "<span class='boldannounce'>You are the Eldritch Cultist!</span>")
+	to_chat(owner, "<B>The old ones gave you these tasks to fulfill:</B>")
+	owner.announce_objectives()
+	to_chat(owner, "Your magic uses souls of the dead. You are very weak at first,")
+	to_chat(owner, "but the more souls you reap the more powerful you become.")
+	to_chat(owner, "You can choose a specific old one you can worship:")
+	to_chat(owner, "Rust - decay and destruction, passive destructive abilities")
+	to_chat(owner, "Flesh - life and necrosis, allows you to ressurect the dead and summon deadly beasts.")
+	to_chat(owner, "Ash - shadows and secrets, stealth and movement based abilties.")
+	to_chat(owner,"<B>Remember your power comes with souls!</B>")
```

Comment:
```suggestion
	to_chat(owner,"<B>Remember that your power comes from souls!</B>")
```
aight, imma stop with the grammar checking

Reply:
thanks a lot!